app/models/decidim/participatory_documents/annotation.rb

```diff
@@ -4,6 +4,10 @@ module Decidim
   module ParticipatoryDocuments
     class Annotation < ApplicationRecord
       belongs_to :zone, class_name: "Decidim::ParticipatoryDocuments::Zone"
```

Comment:
The `create_table` migration says it belongs to a document,
should we change that?

```ruby
  t.belongs_to :document,
                   null: true,
                   index: false
```

Reply:
This PR adds a small change to that in : [db/migrate/20221205093306_change_decidim_participatory_documents_annotations.rb](https://github.com/openpoke/decidim-module-participatory-documents/pull/14/files#diff-094c14c1f46c97e699e58c8ddc12c2ec4ba87e859744eac23d60748e29bc1aff) 

Which will add the Zone as container for annotations. We could drop the document_id column. 

```
add_reference :decidim_participatory_documents_annotations, :zone, index: true, foreign_key: { to_table: :decidim_participatory_documents_zones }
```